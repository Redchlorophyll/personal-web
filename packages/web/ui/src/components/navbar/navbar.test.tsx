import { render } from '@testing-library/react';
import { Navbar } from './navbar';

describe('<Navbar />', () => {
  test('should render properly', () => {
    const { baseElement } = render(<Navbar>Navbar header</Navbar>);

    expect(baseElement).toMatchSnapshot();
    expect(baseElement).toBeTruthy();
  });
});
