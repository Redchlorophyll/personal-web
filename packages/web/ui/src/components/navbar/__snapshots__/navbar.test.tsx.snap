// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Navbar /> should render properly 1`] = `
<body>
  <div>
    <div
      class="w-full h-[67px] bg-black-100 mb-[8vh] md:mb-[15vh] shadow-[0_2px_8px_1px_rgba(0,0,0,0.1)] flex justify-center dark:bg-[#282828] dark:shadow-[0_2px_8px_1px_rgba(154,154,154,0.1)]"
    >
      Navbar header
    </div>
  </div>
</body>
`;
